api/utilities/schemas/user_schema.py

```diff
@@ -0,0 +1,39 @@
+""" 
+User model serializer 
+"""
+
+from marshmallow import fields, post_load
+from api.utilities.schemas.base_schema import AuditableBaseSchema
+from api.utilities.messages.error_messages import SERIALIZATION_ERRORS
+from api.utilities.validators.email_validator import email_validator
+
+from passlib.apps import custom_app_context as pwd_context
+
+
+class UserSchema(AuditableBaseSchema):
+    """
+    user model schema
+    """
+    id = fields.String()
+    first_name = fields.String(
+        load_from='firstname',
+        dump_to='firstname',
+        required=True,
+        error_messages={'required': SERIALIZATION_ERRORS['field_required']})
+    last_name = fields.String(
+        load_from='lastname',
+        dump_to='lastname',
```

Comment:
What case are you loading/dumping to? Your `AuditableBaseSchema` implementation suggests you're using `camelCase`. Be consistent.